host-controller/src/main/java/org/jboss/as/host/controller/embedded/EmbeddedHostControllerBootstrap.java

```diff
@@ -3,25 +3,25 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.wildfly.core.embedded;
+package org.jboss.as.host.controller.embedded;
+
 
-import java.beans.PropertyChangeListener;
 import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.atomic.AtomicReference;
+import java.util.concurrent.Future;
 
 import org.jboss.as.controller.ControlledProcessState;
-import org.jboss.as.controller.ProcessStateNotifier;
 import org.jboss.as.controller.ControlledProcessStateService;
 import org.jboss.as.host.controller.HostControllerEnvironment;
 import org.jboss.as.host.controller.HostControllerService;
 import org.jboss.as.host.controller.HostRunningModeControl;
 import org.jboss.as.server.FutureServiceContainer;
 import org.jboss.as.server.jmx.RunningStateJmx;
+import org.jboss.msc.service.ServiceActivator;
 import org.jboss.msc.service.ServiceContainer;
 import org.jboss.msc.service.ServiceTarget;
 
 /**
- * Bootstrap of the Embedded HostController process.
+ * Embedded variant of {@link org.jboss.as.host.controller.HostControllerBootstrap}.  TODO see if these can be unified.
```

Comment:
Not a blocker for this PR, but was this TODO resolved?

Reply:
@jamezp No, we still have duplication, hence that class.

IIRC the work I did on this got the embedded and non-embedded bootstrap code a bit more aligned, but there are still small differences.

Reply:
@jamezp I filed https://issues.redhat.com/browse/WFCORE-7143. I'll update the TODO here because this PR could use fresh CI anyway. But I'll make no change beyond a tweak to this comment, so no need for a re-review.